fastlane/lib/fastlane/actions/slack.rb

```diff
@@ -40,7 +40,7 @@ def self.run(options)
         return [notifier, slack_attachment] if Helper.test? # tests will verify the slack attachments and other properties
 
         begin
-          results = notifier.ping('', icon_url: icon_url, attachments: [slack_attachment])
+          results = notifier.ping('', link_names: options[:link_names], icon_url: icon_url, attachments: [slack_attachment])
```

Comment:
Can you also set the option to a variable above and use that variable here? Since that is what it seems like this action tends to do 😊